content/page/supporters.en.md

```diff
@@ -4,50 +4,34 @@ date: 2018-08-17 17:03:06
 draft: false
 ---
 
-# Our partners
-<div class="row justify-content-md-center">
-    <div class="col-md-4 text-center">
-        <img src="/media/partners/fdroid.svg" width="120px" height="120px" class="mt-3 ml-auto mr-auto"/>
-        <div class="card-body">
-            <h6 class="card-title"><a href="https://f-droid.org/">F-Droid</a></h6>
-        </div>
-    </div>
-    <div class="col-md-4 text-center">
-        <img src="/media/partners/yale-privacy-lab.png" width="120px" height="120px"class="mt-3 ml-auto mr-auto"/>
-        <div class="card-body">
-            <h6 class="card-title"><a href="https://privacylab.yale.edu/">Yale Privacy Lab</a></h6>
-        </div>
-    </div>
-</div>
-
 # Our supporters
 <div class="row justify-content-md-center">
     <div class="col-md-4 text-center">
-        <img src="/media/partners/logo-code-lutin.svg" width="120px" height="120px" class="mt-3 ml-auto mr-auto"/>
+        <img src="/media/partners/logo-code-lutin.svg" width="120px" class="mt-3 ml-auto mr-auto"/>
         <div class="card-body">
             <h6 class="card-title"><a href="https://www.codelutin.com/">Code Lutin</a></h6>
         </div>
     </div>
     <div class="col-md-4 text-center">
-        <img src="/media/partners/codeurs-en-liberte.svg" width="120px" height="120px" class="mt-3 ml-auto mr-auto"/>
+        <img src="/media/partners/codeurs-en-liberte.svg" width="120px" class="mt-3 ml-auto mr-auto"/>
         <div class="card-body">
             <h6 class="card-title"><a href="https://www.codeursenliberté.fr/">Codeurs en liberté</a></h6>
         </div>
     </div>
     <div class="col-md-4 text-center">
-        <img src="/media/partners/gandi.png" width="120px" height="120px" class="mt-3 ml-auto mr-auto"/>
+        <img src="/media/partners/lqdn.svg" width="120px" class="mt-3 ml-auto mr-auto"/>
```

Comment:
Il doit y avoir un soucis, les partenaires ne sont plus dans l'ordre alphabétique comme cela avait été mis en place auparavant (valable pour en et fr)

Reply:
My bad. I modified the 2 pads with the alphabetic order. Thanks !